gcs-fetcher/cmd/gcs-fetcher/main.go

```diff
@@ -59,19 +66,35 @@ func main() {
 		return
 	}
 
+	var stdout, stderr io.Writer
```

Comment:
Is there some reason you can't just do `stdout, stderr := os.Stdout, os.Stderr` (or some variation thereof)?

Reply:
They take the wrong type,  making the `stderr = io.MultiWriter(stderr, f)` assignment later on fail.

Reply:
Oh, in that case, `var stdout, stderr io.Writer = os.Stdout, os.Stderr`

Reply:
Done.